src/annotation/Validator.java

```diff
@@ -0,0 +1,52 @@
+package annotation;
+
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+    public static List<?> validate(Object object) throws IllegalAccessException {
+        List<String> list = new ArrayList<>();
+        Class<?> objectClass = object.getClass();
+
+        Field[] fields = objectClass.getDeclaredFields();
+        for (Field field : fields) {
+            field.setAccessible(true);
```

Comment:
Reflection should not be used to increase accessibility of classes, methods, or fields.

This rule raises an issue when reflection is used to change the visibility of a class, method or field, and when it is used to directly update a field value. Altering or bypassing the accessibility of classes, methods, or fields violates the encapsulation principle and could lead to run-time errors.

This accessibility update should be removed